### PR TITLE
Upgrade Canasta to MediaWiki 1.39

### PR DESCRIPTION
**Pull request by jeffw16**
- Upgrades Canasta to MediaWiki 1.39
- Upgrades Debian to 11.5

Currently targets MediaWiki 1.39.0

### CODE REVIEW

File: _sources/configs/composer.canasta.json [hunk 1]
```diff
@@ -1,17 +1,9 @@
...
-		"mediawiki/semantic-media-wiki": "3.2.3",
-		"mediawiki/semantic-result-formats": "3.2.0",
-		"mediawiki/semantic-breadcrumb-links": "2.0",
-		"mediawiki/semantic-compound-queries": "2.1.0",
-		"mediawiki/semantic-extra-special-properties": "2.1.0",
-		"mediawiki/semantic-scribunto": "2.1.0",
-		"mediawiki/simple-batch-upload": "1.8.2",
-		"mediawiki/bootstrap-components": "4.0.1",
-		"mediawiki/sub-page-list": "1.6.1"
+		"mediawiki/semantic-media-wiki": "^4.0.0"
```
Comment (pastakhov) [inline _sources/configs/composer.canasta.json:6]: Can we use "4.0.0" to install every time we build docker image the same version? I don't think we want it to be updated accidentally, and same for the chameleon skin.

Comment (pastakhov) [reply _sources/configs/composer.canasta.json:6]: "mediawiki/semantic-media-wiki": "4.0.2"

File: Dockerfile [hunk 2]
```diff
@@ -1,10 +1,10 @@
-FROM debian:11.4 as base
+FROM debian:11.5 as base
 
 LABEL maintainers=""
 LABEL org.opencontainers.image.source=https://github.com/CanastaWiki/Canasta
 
-ENV MW_VERSION=REL1_35 \
-	MW_CORE_VERSION=1.35.8 \
+ENV MW_VERSION=REL1_39 \
+	MW_CORE_VERSION=1.39.0 \
```
Comment (pastakhov) [inline Dockerfile:7]: 1.39.1 was released

File: _sources/configs/composer.canasta.json [hunk 3]
```diff
@@ -1,17 +1,9 @@
 {
 	"require": {
-		"elasticsearch/elasticsearch": "6.7.2",
+		"elasticsearch/elasticsearch": "^7.11.0",
 		"mediawiki/chameleon-skin": "^3.4.1",
 		"mediawiki/maps": "8.0.0",
```
Comment (pastakhov) [inline _sources/configs/composer.canasta.json:5]: "mediawiki/maps": "10.0.0",    (8.0.0  does not work with 1.39)

File: _sources/configs/composer.canasta.json [hunk 4]
```diff
@@ -1,17 +1,9 @@
 {
 	"require": {
-		"elasticsearch/elasticsearch": "6.7.2",
+		"elasticsearch/elasticsearch": "^7.11.0",
```
[the remark this answers is not in the document]
Comment (tosfos) [reply _sources/configs/composer.canasta.json:3]: Indeed. See https://phabricator.wikimedia.org/T276854

File: _sources/configs/composer.canasta.json (hunk 1 above)
Comment (jeffw16) [reply _sources/configs/composer.canasta.json:7]: Got it

File: _sources/configs/composer.canasta.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (jeffw16) [reply _sources/configs/composer.canasta.json:3]: Removed!

File: Dockerfile (hunk 2 above)
Comment (jeffw16) [reply Dockerfile:7]: Fixed

File: _sources/configs/composer.canasta.json (hunk 3 above)
Comment (jeffw16) [reply _sources/configs/composer.canasta.json:5]: Changed, thanks!

**Review by yaronkoren — APPROVED** [review]

**Pull request merged**